test/rest.browser.test.js

```diff
@@ -30,7 +30,7 @@ describe('strong-remoting-rest', function() {
 
   // setup
   beforeEach(function() {
-    objects = RemoteObjects.create();
+    objects = RemoteObjects.create({cors: false});
```

Comment:
Is it necessary to set CORS in all the tests? Should these tests not use the default settings?